apps/ios/Shared/Views/Chat/ChatInfoView.swift

```diff
@@ -649,16 +639,41 @@ struct ChatInfoView: View {
     }
 }
 
+func syncContactConnection(_ contact: Contact, _ connectionStats: Binding<ConnectionStats?>, force: Bool, showAlert: (SomeAlert) -> Void) async -> Bool {
+    do {
+        let stats = try apiSyncContactRatchet(contact.apiId, force)
+        await MainActor.run {
+            connectionStats.wrappedValue = stats
+            ChatModel.shared.updateContactConnectionStats(contact, stats)
+        }
+        return true
+    } catch let error {
+        logger.error("syncContactConnection apiSyncContactRatchet error: \(responseError(error))")
+        let a = getErrorAlert(error, "Error synchronizing connection")
+        await MainActor.run {
+            showAlert(
+                SomeAlert(
+                    alert: mkAlert(title: a.title, message: a.message),
+                    id: "syncContactConnection error"
+                )
+            )
+        }
```

Comment:
using declarative alert because getErrorAlert returns ErrorAlert that has LocalizedStringKeys, and imperative alert requires Strings